src/app/util/attribute-storage.cpp

```diff
@@ -878,6 +878,51 @@ static uint16_t findIndexFromEndpoint(EndpointId endpoint, bool ignoreDisabledEn
     return kEmberInvalidEndpointIndex;
 }
 
+uint16_t emberAfGetClusterServerEndpointIndex(EndpointId endpoint, ClusterId cluster, uint16_t fixedClusterServerEndpointCount)
+{
+    uint16_t epIndex = findIndexFromEndpoint(endpoint, true /*ignoreDisabledEndpoints*/);
```

Comment:
We should probably either fail out or die if fixedClusterServerEndpointCount > FIXED_ENDPOINT_COUNT.

Reply:
Agreed, adding it